proton

```diff
@@ -792,7 +792,7 @@ class CompatData:
                 os.symlink("/", self.prefix_dir + "/dosdevices/z:")
 
             # collect configuration info
-            steamdir = os.environ["STEAM_COMPAT_CLIENT_INSTALL_PATH"]
+            steamdir = os.environ.get("STEAM_COMPAT_CLIENT_INSTALL_PATH", "")
```

Comment:
We depend on Steam setting up this variable and using `[]` will make this fail in a clear way if this ever regresses in Steam.

By doing `.get()` the way you do here we would silence this but also copying of the steam DLLs would silently fail.